frontend/src/modules/layout/components/workspace-dropdown.vue

```diff
@@ -46,7 +46,13 @@
                     {{ getPlan(currentTenant.plan) }}
                     <span
```

Comment:
If the only action this badge will be performing is router.push, maybe we could use `<router-link>` instead and drop the import/initialization related to the router

Reply:
You're absolutely right, I didn't use it because I thought I couldn't stop the click propagation with it. Just saw that I can the same way, so updated!

Thanks for the suggestion 🙏